resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -458,6 +458,41 @@ class MesosCoarseGrainedSchedulerBackendSuite extends SparkFunSuite
     backend.start()
   }
 
+  test("checkpointing is enabled in created scheduled driver") {
+    val checkpointExpected = true
+    initializeSparkConf(Map(CHECKPOINT.key -> checkpointExpected.toString))
+    sc = new SparkContext(sparkConf)
+
+    val taskScheduler = mock[TaskSchedulerImpl]
+    when(taskScheduler.sc).thenReturn(sc)
+
+    val driver = mock[SchedulerDriver]
+    when(driver.start()).thenReturn(Protos.Status.DRIVER_RUNNING)
+
+    val securityManager = mock[SecurityManager]
+
+    val backend = new MesosCoarseGrainedSchedulerBackend(
+      taskScheduler, sc, "master", securityManager) {
+      override protected def createSchedulerDriver(
+                                                    masterUrl: String,
+                                                    scheduler: Scheduler,
+                                                    sparkUser: String,
+                                                    appName: String,
+                                                    conf: SparkConf,
+                                                    webuiUrl: Option[String] = None,
+                                                    checkpoint: Option[Boolean] = None,
+                                                    failoverTimeout: Option[Double] = None,
+                                                    frameworkId: Option[String] = None): SchedulerDriver = {
```

Comment:
nit: does the formatting need to be this far out? Other methods are a bit easier to read with it being closer in.

Reply:
fixed